doc/crds/ip-reconciler-job.yaml

```diff
@@ -7,9 +7,13 @@ metadata:
     tier: node
     app: whereabouts
 spec:
+  concurrencyPolicy: Forbid
+  failedJobsHistoryLimit: 0
```

Comment:
not exactly sure we want to delete the failed pods. What if we want to debug them ?

Reply:
I like the idea of maybe setting this to `1`? That way we can see one failure so we can debug it?

Reply:
That's OK, but afaiu, 1 is the default. I can just remove the entry altogether.

https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#cronjobspec-v1beta1-batch